bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,155 @@
+# Solarwatt Binding
+
+Binding to query a [solarwatt](https://www.solarwatt.de/) [energy manager](https://www.solarwatt.de/energie-management/energymanager) and read the values of all attached devices.
+
+All supported values and devices were discovered while playing with my own energy manager.
+
+
```

Comment:
I am pretty sure double empty lines is reported in the code analysis. After you have compiled, go into the target folder and there will be a code-analysis folder. In there open the report and fix up any of the changes it suggests.

Reply:
For README.md it only reported `File does not end with a newline.`

Reply:
All but one finding from code-analysis is fixed.
Remaining warning is about empty if statement, which I would like to keep at that place.